interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/in/RequestDaoServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.moj.sdt.interceptors.in;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.moj.sdt.dao.ServiceRequestDao;
+import uk.gov.moj.sdt.interceptors.service.RequestDaoService;
+import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
+import uk.gov.moj.sdt.utils.SdtContext;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+@ExtendWith(MockitoExtension.class)
+class RequestDaoServiceTest extends AbstractSdtUnitTestBase {
+
+    @Mock
+    ServiceRequestDao mockServiceRequestDao;
+
+    RequestDaoService requestDaoService;
+
+    @Override
+    protected void setUpLocalTests() {
+        requestDaoService = new RequestDaoService(mockServiceRequestDao);
+    }
+
+    /**
+     * Test that the process correctly works via mocked out extensions.
+     */
+    @Test
+    void testHandleMessage() {
+
+        // Setup the raw XML as if the XmlInboundInterceptor had run.
+        final String xml =
+                this.convertStreamToString(ServiceRequestInboundInterceptorTest.class
+                        .getResourceAsStream("inRequest.xml"));
+        SdtContext.getContext().setRawInXml(xml);
+
+        requestDaoService.persistRequest();
+    }
```

Comment:
This test needs to check something, otherwise it doesn't really have any benefit.  Easiest thing to check is that the persist() method was called on the mock ServiceRequestDao object, i.e. `verify(mockServiceRequestDao).persist(any(ISeviceRequest.class));`